libs/eck-operator/config.jsonnet

```diff
@@ -0,0 +1,18 @@
+# libs/eck-operator/config.jsonnet
+local config = import 'jsonnet/config.jsonnet';
+
+local versions = ['1.7.0'];
+
+config.new(
+  name='eck-operator',
+  specs=[
+    {
+      output: version,
+      openapi: 'http://localhost:8001/openapi/v2',
+      prefix: '^co\\.elastic\\.k8s\\..*',
+      crds: ['https://raw.githubusercontent.com/elastic/cloud-on-k8s/%s/config/crds/v1/all-crds.yaml' % version],
+      localName: 'eck-operator',
+    }
+    for version in versions
```

Comment:
I recently realized that this won't work for CRD based generation ( it does for k8s which uses swagger file directly ) because all CRDS will be loaded at once ( with the last one winning ) in a k3s cluster and then each of the specs wlil be rendered essentially with the same generated output 

see https://github.com/jsonnet-libs/k8s/blob/master/scripts/gen.sh#L14

Reply:
https://github.com/jsonnet-libs/k8s/issues/60